src/ui/utils/environment.ts

```diff
@@ -128,7 +131,9 @@ export function updateUrlWithParams(params: Record<string, string>) {
   const url = new URL(window.location.toString());
 
   Object.entries(params).forEach(([key, value]) => {
-    url.searchParams.set(key, value);
+    if (value !== undefined && value !== null) {
```

Comment:
It's important to get this check right. `point=0` is a valid point, even though `0` is falsey. But we (almost certainly) never want to put the values `undefined` or `null` directly into a URL.